src/key_file.rs

```diff
@@ -70,4 +70,21 @@ impl KeyFile {
             from_glib_full(ret)
         }
     }
+
+    pub fn get_boolean(&self, group_name: &str, key: &str) -> Result<bool, Error> {
+        unsafe {
+            let mut error = ptr::null_mut();
+            let ret = ffi::g_key_file_get_boolean(self.to_glib_none().0, group_name.to_glib_none().0, key.to_glib_none().0, &mut error);
+            if error.is_null() { Ok(from_glib(ret)) } else { Err(from_glib_full(error)) }
```

Comment:
As I understand from https://github.com/gtk-rs/gir/issues/580 this function better use `g_key_file_get_value` internally?

Reply:
Or this if need 3th case if g_key_file_get_boolean return false and error = null for `false` value

Reply:
I'm not sure what you mean.  The problem that I'm fixing is that GIR assumed that the `gboolean` return value from g_key_file_get_boolean was redundant and useless, and so it generated a function that dropped the `gboolean` return value and returned `Result<(), Error>` instead.  This is an OK assumption because it's often the right thing to do for functions that throw and also return `gboolean`.

But in this particular case, the `gboolean` return value is exactly what we want to return back to the caller.

The `&mut error` parameter is still important and still needs to be checked.

It looks good to me.

Reply:
Oops, I missed that if condition changed.
No problem for me now.